src/CMakeLists.txt

```
@@ -57,59 +57,109 @@ set_target_properties(zoe PROPERTIES
 
 target_include_directories(zoe
 	PUBLIC $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include>
-	PUBLIC $<INSTALL_INTERFACE:../include>
+	PUBLIC $<INSTALL_INTERFACE:include>
 )
 
 # CURL
 find_package(CURL REQUIRED)
-if(ZOE_BUILD_SHARED_LIBS)	
-	target_link_libraries(zoe PRIVATE ${CURL_LIBRARIES})
-else()
-	target_link_libraries(zoe PUBLIC ${CURL_LIBRARIES})
-endif()
-
-target_include_directories(zoe PRIVATE ${CURL_INCLUDE_DIRS})
+target_link_libraries(zoe PRIVATE CURL::libcurl)
+target_compile_definitions(zoe PRIVATE WITH_LIBCURL)
 
 # OpenSSL
 find_package(OpenSSL)
+
 if(OpenSSL_FOUND)
+	target_link_libraries(zoe PRIVATE OpenSSL::SSL)
 	target_compile_definitions(zoe PRIVATE WITH_OPENSSL)
+endif()
+
+if(ZOE_BUILD_SHARED_LIBS)
+	set(ZOE_PC_NAME "zoe")
+	set(ZOE_CONFLICTS "zoe-static")
+else()
+	set(ZOE_PC_NAME "zoe-static")
+	set(ZOE_CONFLICTS "zoe")
+endif()
+
+if(OpenSSL_FOUND)
+	set(ZOE_REQUIRES "libssl libcurl")
+else()
+	set(ZOE_REQUIRES "libcurl")
+endif()
```

Nitpick: With more optional dependencies, the following pattern scales better:
```suggestion
set(ZOE_REQUIRES "libcurl")
if(OpenSSL_FOUND)
	string(APPEND ZOE_REQUIRES " libssl")
endif()
```